airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/manifest.yaml

```diff
@@ -0,0 +1,2966 @@
+version: 6.48.16
+
+type: DeclarativeSource
+
+check:
+  type: CheckDynamicStream
+  stream_count: 1
+  use_check_availability: false
+
+definitions:
+  streams:
+    google_analytics_stream:
```

Comment:
nit lets call this `google_analytics_stream_template` just so we know its not a "real" stream

Reply:
Updated to `google_analytics_stream_template`